src/components/organisms/about/about.html

```diff
@@ -0,0 +1,19 @@
+<section class="about-container">
```

Comment:
左右の余白を調整するのにBootStrapのcontainerを使うように指定があるので修正お願いします
https://getbootstrap.jp/docs/5.0/layout/containers/
```suggestion
<section class="about-container container">
```